docker/Dockerfile.ubuntu.xenial

```diff
@@ -0,0 +1,10 @@
+# Dockerfile for building an image with ubuntu xenial
+FROM sociomantictsunami/base
+MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
+
+WORKDIR /home
```

Comment:
I wonder why you need to set a workdir at all.

Reply:
Hmm, isn't `root` the default user in the images, which means that `$HOME` is  `/root`

Reply:
Cf, https://docs.docker.com/engine/reference/run/#user and https://docs.docker.com/engine/reference/run/#workdir

Reply:
When building the image it is root. But the question is why you need to know which is the work directory? Are you writing any files to it?

I think you are mixing up stuff, this is image creating, for running you just pass `-u`. The `USER` and `WORKDIR` commands make (more?) sense when using docker to actually deploy stuff, but for testing you should just run `docker run -u user -w workdir` if you want to set it to anything in particular, you don't have to do anything at the image creation unless you need it to actually create the image.

Reply:
> Are you writing any files to it?

Only writing the init script `ubuntu_xenial_image_run.sh` to `/usr/local/bin/`

> if you want to set it to anything in particular, you don't have to do anything at the image creation unless you need it to actually create the image.

Ah, I can set the working directory in the `docker-compose.yml` file, https://docs.docker.com/compose/compose-file/#domainname-hostname-ipc-mac_address-privileged-read_only-shm_size-stdin_open-tty-user-working_dir